markets/bfp-market/test/integration/modules/OrderModule.test.ts

```diff
@@ -104,6 +104,36 @@ describe('OrderModule', () => {
       );
     });
 
+    it('should revert on a too large order', async () => {
```

Comment:
You can make this test a little bit less brittle by calculating the actual min credit then, similar to other tests, generate deposit values that exceed the min credit. Also consider renaming this test to mention that this is testing for min credit